feat: Add ClusterResourceQuota

sarahbx opened the pull request:
Add ClusterResourceQuota for AAQ testing

##### Short description:

##### More details:

##### What this PR does / why we need it:

##### Which issue(s) this PR fixes:

##### Special notes for reviewer:

##### Bug:


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
  - Introduced a new class for managing OpenShift cluster resource quotas with customizable resource limits and namespace selection.
  - Added a constant to improve referencing of OpenShift quotas within the API context.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

sarahbx commented:
/verified

myakove commented:
@sarahbx 
Please create the resource using [class-generator](https://github.com/RedHatQE/openshift-python-wrapper/blob/main/class_generator/README.md)

sarahbx commented:
/verified

myakove commented:
@sarahbx 
Please generate with the latest class-generator version

myakove commented:
/lgtm
any reason why it's draft?

sarahbx commented:
@myakove I had moved it to draft seeing this comment, was going to re-run with latest generator when I had a chance
https://github.com/RedHatQE/openshift-python-wrapper/pull/2355#issuecomment-2744691736
moved back to ready

myakove commented:
You are right, thanks
I missed my own comment :)

myakove commented:
This is still with the old version right?

sarahbx commented:
no worries, yup. will re-run : )

sarahbx commented:
well hello coderabbit  O.o

myakove commented:
/verified
